lobby/src/main/java/games/strategy/engine/lobby/server/LobbyRunner.java

```diff
@@ -0,0 +1,29 @@
+package games.strategy.engine.lobby.server;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import games.strategy.sound.ClipPlayer;
+
+/**
+ * A 'main' class to launch the lobby server.
+ */
+public class LobbyRunner {
+  private static final Logger logger = Logger.getLogger(LobbyServer.class.getName());
```

Comment:
Should this logger be based off of `LobbyRunner.class.getName()`?

Reply:
Yes, copy/paste error on my part - good catch.

https://github.com/triplea-game/triplea/pull/3287 would fix: 81935f7